frontend/static/scripts/projects_scripts.js

```diff
@@ -0,0 +1,87 @@
+/* projects_scripts.js
+ * This file runs on top of the software overview page (templates/project/project_index.html).
+ * It is served as-is, so please keep everything ES5 compatible.
+*/
+
+/*** Copyright 2013 Teun Duynstee Licensed under the Apache License, Version 2.0 ***/
+var firstBy=function(){function n(n){return n}function t(n){return"string"==typeof n?n.toLowerCase():n}function r(r,e){if(e="number"==typeof e?{direction:e}:e||{},"function"!=typeof r){var i=r;r=function(n){return n[i]?n[i]:""}}if(1===r.length){var u=r,o=e.ignoreCase?t:n;r=function(n,t){return o(u(n))<o(u(t))?-1:o(u(n))>o(u(t))?1:0}}return-1===e.direction?function(n,t){return-r(n,t)}:r}function e(n,t){var i="function"==typeof this&&this,u=r(n,t),o=i?function(n,t){return i(n,t)||u(n,t)}:u;return o.thenBy=e,o}return e}();
+/*** https://github.com/component/debounce ***/
+function debounce(n,l,u){function t(){var c=Date.now()-r;c<l&&c>=0?e=setTimeout(t,l-c):(e=null,u||(i=n.apply(o,a),o=a=null))}var e,a,o,r,i;null==l&&(l=100);var c=function(){o=this,a=arguments,r=Date.now();var c=u&&!e;return e||(e=setTimeout(t,l)),c&&(i=n.apply(o,a),o=a=null),i};return c.clear=function(){e&&(clearTimeout(e),e=null)},c.flush=function(){e&&(i=n.apply(o,a),o=a=null,clearTimeout(e),e=null)},c}
+
+function initOverview(projectsData) {
+    var device = {
+        phone: 'phone',
+        tablet: 'tablet',
+        desktop: 'desktop'
+    };
+
+    function getDevice() {
+        return window.innerWidth > 1000 ? device.desktop : (window.innerWidth > 700 ? device.tablet : device.phone);
```

Comment:
Nested ternary ...nice :muscle: 

At some point, we should revisit this file and software_scripts.js to increase the readability a bit and also apply some best practices. Possibly around the same time we'll be working on #631 and #632.

Reply:
> Another suggestion: I would prefer to
> 
> ```
> mkdir project_index
> git mv project/project_index.html project_index/
> mkdir software_index
> git mv index_template.html software_index/software_index.html
> ```

Could you make a separate issue for this? This has nothing to do with the issue this PR is solving.

Reply:
made #678